src/views/layouts/form.blade.php

```diff
@@ -341,4 +341,14 @@
 
 	</div>
 	
+	<script src="http://cdnjs.cloudflare.com/ajax/libs/summernote/0.8.11/summernote.js"></script>
+	<script>
+        var textareas = document.getElementsByTagName("textarea")
+        for (var i = 0; i < textareas.length; i++) {
+            textareas[i].className += 'summernote';
+        }
+        $(document).ready(function() {
+            $('.summernote').summernote();
+        });
+	</script>
```

Comment:
@ayberkersoy the '.summernote' class is not required you could simply do something like:

```
       $(document).ready(function() {
            $('.station-form textarea').summernote();
        });
```

Reply:
Also please let me know how all text areas where being saved in database? Was it html? Or are we using a js framework to process markdown? @ayberkersoy